src/PolyhedralGeometry/SubdivisionOfPoints/constructors.jl

```diff
@@ -47,11 +47,12 @@ Subdivision of points in ambient dimension 3
 ```
 """
 function subdivision_of_points(::Type{T}, points::AbstractCollection[PointVector], cells::IncidenceMatrix) where T<:scalar_types
-   arr = @Polymake.convert_to Array{Set{Int}} Polymake.common.rows(cells)
-   SubdivisionOfPoints{T}(Polymake.fan.SubdivisionOfPoints{scalar_type_to_polymake[T]}(
-      POINTS = homogenize(points,1),
-      MAXIMAL_CELLS = arr,
-   ))
+  @assert size(points)[1] == ncols(cells) "Number of points must be the same as columns of IncidenceMatrix"
```

Comment:
```suggestion
  @req size(points)[1] == ncols(cells) "Number of points must be the same as columns of IncidenceMatrix"
```
Maybe these checks should be `@req` instead of assert? (to make sure that they are always executed)